tests/pages/title-search.js

```diff
@@ -27,9 +27,11 @@ export default {
   },
 
   get $searchField() {
+    return $('[data-test-title-search-field]').find('input[name="search"]');
+  },
+  get $otherSearchTypeSearchField() {
```

Comment:
In `pages/package-search` we left this as `$searchField` and called the other one `$titleSearchField`. We should keep these consistent and decide which way we want them to be.

Reply:
could add a packageSearchField and a providerSearchField (these would be the same element)